docs/src/content/docs/ko/guides/sidebar.mdx

```diff
@@ -434,6 +434,72 @@ starlight({
 	]}
 />
 
+### 자동 생성 링크의 사용자 정의 HTML 속성
+
+`autogenerate` 구성의 `attrs` 속성을 정의하여 [자동 생성 그룹](#자동-생성-그룹)에 있는 모든 링크의 HTML 속성을 사용자 정의합니다.
+개별 페이지에서는 [`sidebar.attrs` 프런트매터 필드](/ko/reference/frontmatter/#attrs)를 사용하여 사용자 정의 속성을 지정할 수 있으며, 이는 `autogenerate.attrs` 구성과 병합됩니다.
+
+예를 들어, 다음과 같은 구성을 사용할 수 있습니다.
+
+```js {9}
+starlight({
+	sidebar: [
+		{
+			label: '별자리',
+			autogenerate: {
+				// 'constellations' 디렉터리에 대한 링크 그룹을 자동으로 생성합니다.
+				directory: 'constellations',
+				// 이 그룹의 모든 링크 라벨을 이탤릭체로 표시합니다.
+				attrs: { style: 'font-style: italic' },
+			},
+		},
+	],
+});
+```
+
+그리고 파일 구조는 다음과 같습니다.
+
+<FileTree>
+
+- src/
+  - content/
+    - docs/
+      - constellations/
+        - carina.md
+        - centaurus.md
+        - seasonal/
+          - andromeda.md
+
+</FileTree>
+
+다음과 같이 모든 자동 생성 링크가 이탤릭체로 표시된 사이드바가 생성됩니다.
+
+<SidebarPreview
+	config={[
+		{
+			label: '별자리',
+			items: [
+				{ label: 'Carina', link: '', attrs: { style: 'font-style: italic' } },
```

Comment:
```suggestion
				{ label: '용골자리', link: '', attrs: { style: 'font-style: italic' } },
```

In this case, the label would be the frontmatter title so translated, just like the ["Autogenerated groups" section](https://starlight.astro.build/ko/guides/sidebar/#%EC%9E%90%EB%8F%99-%EC%83%9D%EC%84%B1-%EA%B7%B8%EB%A3%B9) which is basically the same example.